src/navigation/navigators/AppNavigator.js

```diff
@@ -1,11 +1,11 @@
 import { createAppContainer, createSwitchNavigator } from "react-navigation";
 import RegistrationNavigator from "./RegistrationNavigator";
 import MainPageNavigator from "./MainPageNavigator";
-import { LanguageSelect } from "../../screens";
+import IntroductionNavigator from "./IntroductionNavigator";
 
 const AppSwitchNavigator = createSwitchNavigator({
-  languageSelect: {
-    screen: LanguageSelect
+  introduction: {
+    screen: IntroductionNavigator
```

Comment:
why remove LanguageSelect? 
Also introduction screens doesn't appear on language screen design, so I think IntroductionNavigator should be moved to RegistrationNavigator.js.

Reply:
According to designs and Denis Kargin clarification back button on first introduction screen should navigate to language selection also introduction screens should be displayed once like language screen by first app open.